### PR TITLE
docs: translate pricing page (part3)

### PR DESCRIPTION
**Pull request by d-bytebase**

### CODE REVIEW

**Review by boojack — APPROVED** [review]
LGTM

File: locales/zh.json [hunk 1]
```diff
@@ -69,6 +69,23 @@
...
+      "email-support": "邮件支持",
+      "everything-in-free-plan": "所有免费版的功能",
+      "rbac": "RBAC 权限管理",
+      "gitlab-login": "通过 GitLab 登录",
+      "sql-check-anomaly-detection": "高阶 SQL 检查和异常检测",
+      "review-and-backup-policy": "审核和备份策略",
+      "sla": "SLA 支持",
+      "everything-in-team-plan": "所有团队版的功能",
+      "multi-tenancy": "多地域/多租户部署",
+      "more-to-come": "更多功能尽情期待..."
```
Comment (boojack) [inline locales/zh.json:88]:
```suggestion
      "more-to-come": "更多功能尽请期待..."
```

**Review by boojack — APPROVED** [review]

**Review by ecmadao — APPROVED** [review]

File: locales/zh.json (hunk 1 above)
Comment (boojack) [reply locales/zh.json]: It should be "敬请期待" :(

Comment (d-bytebase) [reply locales/zh.json]: Will fix in a follow-up PR

**Pull request merged**